cloudfoundry-operations/src/main/lombok/org/cloudfoundry/operations/quotas/Quota.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2013-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.cloudfoundry.operations.quotas;
+
+import lombok.Builder;
+import lombok.Data;
+
+/**
+ * A Cloud Foundry Quota
+ */
+@Data
+public final class Quota {
+
+    /**
+     * The allow paid service plans flag
+     *
+     * @param allowPaidServicePlans the allow paid service plans flag
+     * @return the allow paid service plans flag
+     */
+    private final Boolean allowPaidServicePlans;
+
+    /**
+     * The application instance limit
+     *
+     * @param applicationInstanceLimit the application instance limit
+     * @return applicationInstanceLimit
+     */
+    private final Integer applicationInstanceLimit;
```

Comment:
@antechrestos This one is interesting.  It's not in the CLI output, but is probably worth having.  Wonder if that constitutes a bug in the CLI.